test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -123,6 +123,30 @@
         }
     }
 
+    Context "ConvertTo-CliXml" {
+        BeforeAll {
+            $gpsList = Get-Process pwsh
+            $gps = $gpsList | Select-Object -First 1
+            $filePath = Join-Path $subFilePath 'gps.xml'
+        }
+
+        AfterEach {
+            Remove-Item $filePath -Force -ErrorAction SilentlyContinue
+        }
+
+        It "converts an object to string as expected" {
```

Comment:
I don't think so, not without making it uglier.  Because the difference in the two tests isn't a variable or parameter, it's function param vs pipeline.   That said, can move some of the duplicate code to the BeforeAll, should make it more clear what is really being tested